internal/workflow.go

```diff
@@ -1457,9 +1456,7 @@ func convertRetryPolicy(retryPolicy *RetryPolicy) *commonpb.RetryPolicy {
 	if retryPolicy == nil {
 		return nil
 	}
-	if retryPolicy.BackoffCoefficient == 0 {
-		retryPolicy.BackoffCoefficient = backoff.DefaultBackoffCoefficient
```

Comment:
`backoff.DefaultBackoffCoefficient` is it still used anywhere?

Reply:
yes

Reply:
NewExponentialRetryPolicy